core-bundle/contao/templates/twig/backend/data_container/operations.html.twig

```diff
@@ -1,20 +1,43 @@
-{% for operation in operations %}
-    {{ _self.operation(operation) }}
-{% endfor %}
+{% set has_submenu = false %}
+<nav>
+    <ul>
+        {% for operation in operations %}
+            {% if has_primary and not operation.primary|default(false) %}
+                {% set has_submenu = true %}
+            {% else %}
+                {{ _self.operation(operation) }}
+            {% endif %}
+        {% endfor %}
+        {% if has_submenu or not has_primary %}
+            <li class="operations-menu-container" data-contao--operations-menu-target="container">
+                {% if has_primary %}
+                    <button title="More" data-contao--operations-menu-target="controller">
```

Comment:
`title="More"` ?

Reply:
Fixed in d0c9057aac438624a47bca783fea63202c261c19